packages/vm/test/api/EIPs/eip-2935-historical-block-hashes.spec.ts

```diff
@@ -211,9 +211,9 @@ describe('EIP 2935: historical block hashes', () => {
     })
     it('should ensure blocks older than 256 blocks can be retrieved from the history contract', async () => {
       // Test: build a chain with 256+ blocks and then retrieve BLOCKHASH of the genesis block and block 1
-      const blocksActivation = 256 // This ensures that block 0 - 255 all get stored into the hash contract
+      const blocksActivation = 256 // This ensures that only block 255 gets stored into the hash contract
       // More than blocks activation to build, so we can ensure that we can also retrieve block 0 or block 1 hash at block 300
-      const blocksToBuild = 300
+      const blocksToBuild = 500
```

Comment:
Do we now need 500 blocks, or can we also "just" create a few blocks (say 5)? We only need to test that the last blockhash is put into the contract at fork activation time.

Reply:
yes the test tests various scenarios of blockhash as well as the contract code windows as well as activation time